sklego/transformers.py

```diff
@@ -2,28 +2,27 @@
 from sklearn.utils import check_array, check_X_y
 from sklearn.utils.validation import FLOAT_DTYPES, check_random_state, check_is_fitted
 
+from sklego.common import TransformerSelectorMixin
 
-class RandomAdder(TransformerMixin, BaseEstimator):
+
```

Comment:
do we need to import both our transformer mixin as well as the one from sklearn?

Reply:
I let the TransformerMixin not inherit from anything, so currently you need to inherit them both. My thinking here was that you could have other mixins as well and still be flexible with the MRO as a user inheriting from them

Reply:
I don't mind that idea but then we need an argument that this will happen in the future. Can you give more examples of Mixins?

Reply:
Good question, I'm not completely sure. Maybe some kind of LoggerMixin or CachingMixin. Main thing is expectation wise that Mixins typically only inherit from object (same as the sklearn mixins e.g. TransformerMixin) and it would feel odd to let RandomAdder only inherit from the mixin and then let the mixin inherit from BaseEstimator.

Other option would be to just inherit from this Mixin and rename it to something without mixin but regular inheritance doesn't seem like the right solution here

Reply:
After a night of sleep: I think it makes sense to combine them, can't think of a use case where you would need it any other way